### PR TITLE
Kafka and nats executer

### PR DESCRIPTION
**Pull request by yvanoers**
New PR to replace #686, updated to master and cleaned up.

### CODE REVIEW

**Comment by cobolbaby** [comment]
Could you resolve the version conflict? @yvanoers

**Comment by yvanoers** [comment]
Closing and reopening to trigger tests.

**Comment by yvanoers** [comment]
@cobolbaby  Voila.

**Comment by cobolbaby** [comment]
Thanks. @yvanoers

**Review by victorcoder — APPROVED** [review]
LGTM but I miss documentation and goreleaser entries here

**Comment by yvanoers** [comment]
@victorcoder Ok, how's this?

File: .goreleaser.yml [hunk 1]
```diff
@@ -34,6 +34,16 @@ builds:
...
 
+  - <<: *xbuild
+    main: ./builtin/bins/dkron-executor-nats/
+    id: dkron-executor-nats
+    binary: dkron-executor-nats
+
+  - <<: *xbuild
+    main: ./builtin/bins/dkron-executor-kafka/
+    id: dkron-executor-kafka
+    binary: dkron-executor-nats
```
Comment (victorcoder) [inline .goreleaser.yml:45]:
```suggestion
    binary: dkron-executor-kafka
```

**Review by victorcoder — APPROVED** [review]

**Review by victorcoder — APPROVED** [review]
Fixed a typo, great docs @yvanoers, thank you!

**Comment by yvanoers** [comment]
> Fixed a typo
Thanks @victorcoder !

**Pull request merged**